CHANGELOG.rst

```diff
@@ -4,6 +4,23 @@ Grafana Collection Release Notes
 
 .. contents:: Topics
 
+v1.9.0
+======
+
+Major Changes
+-------------
+
+- grafana_silence ; adding new module to create and delete silences through the API
```

Comment:
@rndmh3ro should this be changed to `minor_change`?

Reply:
yes.